client/src/modules/dashboard/Venues/pages/VenuePage.tsx

```diff
@@ -1,21 +1,43 @@
-import { Heading, Text } from '@chakra-ui/layout';
+import { Button, Heading, Text } from '@chakra-ui/react';
 import NextError from 'next/error';
 import React, { ReactElement } from 'react';
 import { Link } from 'chakra-next-link';
+import { useConfirmDelete } from 'chakra-confirm';
+import { useRouter } from 'next/router';
 
 import { Card } from '../../../../components/Card';
 import ProgressCardContent from '../../../../components/ProgressCardContent';
-import { useVenueQuery } from '../../../../generated/graphql';
+import {
+  useVenueQuery,
+  useDeleteVenueMutation,
+} from '../../../../generated/graphql';
 import { useParam } from '../../../../hooks/useParam';
 import getLocationString from '../../../../util/getLocationString';
 import styles from '../../../../styles/Page.module.css';
 import { DashboardLoading } from '../../shared/components/DashboardLoading';
 import { EventList } from '../../shared/components/EventList';
 import { DashboardLayout } from '../../shared/components/DashboardLayout';
 import { NextPageWithLayout } from '../../../../pages/_app';
+import { DASHBOARD_VENUE, DASHBOARD_VENUES } from '../graphql/queries';
+import { DASHBOARD_CHAPTER } from '../../../dashboard/Chapters/graphql/queries';
+import {
+  DASHBOARD_EVENT,
+  DASHBOARD_EVENTS,
+} from '../../../dashboard/Events/graphql/queries';
 
 export const VenuePage: NextPageWithLayout = () => {
   const { param: venueId } = useParam('id');
+  const confirmDelete = useConfirmDelete();
+  const [deleteVenue] = useDeleteVenueMutation({
+    refetchQueries: [
+      { query: DASHBOARD_VENUE },
+      { query: DASHBOARD_VENUES },
+      { query: DASHBOARD_CHAPTER },
+      { query: DASHBOARD_EVENT },
+      { query: DASHBOARD_EVENTS },
+    ],
+  });
```

Comment:
```suggestion
  const [deleteVenue] = useDeleteVenueMutation();
```
`DASHBOARD_CHAPTER` will need to get `chapterId` variable, that will be easier to do later. Similarly is with `DASHBOARD_VENUE` and `DASHBOARD_EVENT`, the first one can be dropped though (see below).